src/dist/replication/common/replication_common.h

```diff
@@ -151,6 +153,13 @@ class backup_restore_constant
     static const std::string SKIP_BAD_PARTITION;
 };
 
+class bulk_load_constant
```

Comment:
I think you can change the type of `bulk_load_constant` from class to struct, because the member variables are all public.

Reply:
I used class here is just like `replica_envs` class, I would like to organize const varieties who used in a same function in a namespace.